app/javascript/js/controllers/action_controller.js

```diff
@@ -2,10 +2,12 @@ import { Controller } from '@hotwired/stimulus'
 import { castBoolean } from '../helpers/cast_boolean'
 
 export default class extends Controller {
-  static targets = ['controllerDiv', 'resourceIds', 'form']
+  static targets = ['controllerDiv', 'resourceIds', 'form', 'selectedQuery', 'selectedAll']
```

Comment:
Let's make the name a bit more obvious.

```suggestion
  static targets = ['controllerDiv', 'resourceIds', 'form', 'selectedAllQuery', 'selectedAll']
```

Reply:
sure, fixing this on next commit